### DISCUSSION TITLE
Add link to publications/make the title clickable to reach publication

### DISCUSSION THREAD

**Discussion by AlvaMarkelius** [alshedivat/al-folio#1939] [Q&A] [answered]
Hi, I wonder if it is possible to make the title in the publications list clickable to reach the relevant publication? Not a pdf but rather the springer/arxiv etc page? I have tried as much I can to implement this so I wonder if anyone have succeeded in doing so or know how to do this?
Thank you!

**Comment by george-gca** [comment, marked as answer]
Currently this information can be accessed via a `html` button in the publication. You can check, for example, the publication from 1935 in our [example](https://alshedivat.github.io/al-folio/publications/). Now, to do what you want, just add the [link](https://github.com/alshedivat/al-folio/blob/85f2590a2119ec57063845d628221155044ebea8/_layouts/bib.html#L151) that would appear in the `html` button to the [title](https://github.com/alshedivat/al-folio/blob/85f2590a2119ec57063845d628221155044ebea8/_layouts/bib.html#L35) of the publication.

**Reply by AlvaMarkelius** [reply]
Thank you for your help. I have added my publications in papers.bib in the regular BibTex format. How do I add this information here? Do I add a line for html = **link to publication** or how do i edit this file?

**Reply by george-gca** [reply]
You only have to edit the html code for that page exactly in the link that I provided. You don't have to edit the bibtex entry.
